### ISSUE TITLE
Add `EnableCloudController` flag to the fluxninja plugin

### ISSUE THREAD

**Issue by kklimonda-fn** [fluxninja/aperture#2308]
This flag should default to false, switching it to true should have the following effects:
- DisableLocalOTelPipeline should be toggled to true
- Etcd namespace should be set to an empty value
- Etcd endpoints should be configured to point to the cloud etcd endpoint for the organization

**Comment by kklimonda-fn** [comment]
Settings this flags also needs to configure TLS for the etcd client, but we need to make sure that some TLS configuration (like checking server certificate) can be still configured.

**Comment by krdln** [comment]
@kklimonda-fn Is there any usecase for users disabling checking server certificate? In the "real cloud" we'll always have a valid certificate :shrug: The only place I guess this option could be helpful is in debugging on local setups.

**Comment by kklimonda-fn** [comment]
Yes, local setups (either laptop or GCP DevEnv) and some temporary environments that we use self signed certificates for.
